bluebottle/activities/admin.py

```diff
@@ -257,9 +257,19 @@ def send_impact_reminder_message_link(self, obj):
         )
         return format_html(
             u"<a href='{}'>{}</a>",
-            url, _('Send impact reminder message')
+            url, _('Send reminder message')
         )
-    send_impact_reminder_message.short_description = _('impact reminder')
+    send_impact_reminder_message.short_description = _('Impact Reminder')
+
+    def get_form(self, request, obj=None, **kwargs):
+        kwargs.update({
+            'help_texts': {
+                'send_impact_reminder_message_link': _(
+                    u"Request the activity manager to fill in the impact of his activity."
```

Comment:
Maybe change this:
Request the activity manager to fill in the impact of his activity. 
>>
Request the activity manager to fill in the impact of the activity.

Reply:
@Rolfertjan  ^^